core/gpslocation.cpp

```diff
@@ -612,8 +612,8 @@ void GpsLocation::downloadFromServer()
 
 				struct gpsTracker gt;
 				gt.when = timestamp.toMSecsSinceEpoch() / 1000;
-				gt.latitude.udeg = latitude.toDouble() * 1000000;
-				gt.longitude.udeg = longitude.toDouble() * 1000000;
+				gt.latitude.udeg = lrint(latitude.toDouble() * 1000000);
+				gt.longitude.udeg = lrint(longitude.toDouble() * 1000000);
```

Comment:
@dje29 to silence the warning a regular cast instead of the lrint() should preserve the old behavior.